### PR TITLE
fix(nc-gui): ui updates

### PR DESCRIPTION
**Pull request by reenphygeorge**
## Change Summary

* Left sidebar menu items overflow fixed when window is resized.
* For Email data type - on click anywhere in the cell outside underlined text, only selects cell instead of redirect.
* Alignment change for github button & join cloud button.
* Toast logo & text alignment correction.
* Collaboration modal alignment corrections.
* Darker fonts for Workspace icon.
* Added icon for geometry data type.

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [x] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

### CODE REVIEW

File: packages/nc-gui/components/smartsheet/header/CellIcon.ts [hunk 1]
```diff
@@ -81,6 +81,8 @@ const renderIcon = (column: ColumnType, abstractType: any) => {
     return iconMap.percent
   } else if (isInt(column, abstractType) || isFloat(column, abstractType)) {
     return iconMap.number
+  } else if(column.uidt === UITypes.Geometry) {
```
Comment (wingkwong) [inline packages/nc-gui/components/smartsheet/header/CellIcon.ts:84]: Define `isGeometry` in `packages/nc-gui/utils/cell.ts` and use it here instead. keep the pattern consistent.

File: packages/nc-gui/assets/style.scss [hunk 2]
```diff
@@ -32,6 +32,10 @@
   height: var(--topbar-height) !important;
 }
 
+.anticon-check-circle {
+  @apply !relative top-[-1px] left-0;  
+}
```
Comment (wingkwong) [inline packages/nc-gui/assets/style.scss:37]: I didn't see this class in use. Is it overriding a system class? Please share some steps to tell how to verify it.

File: packages/nc-gui/components/cell/Email.vue [hunk 3]
```diff
@@ -86,9 +86,9 @@ watch(
 
   <span v-else-if="vModel === null && showNull" class="nc-null">NULL</span>
 
-  <a v-else-if="validEmail" class="text-sm underline hover:opacity-75" :href="`mailto:${vModel}`" target="_blank">
+  <nuxt-link v-else-if="validEmail" class="text-sm underline hover:opacity-75 inline-block" :href="`mailto:${vModel}`" target="_blank">
```
Comment (wingkwong) [inline packages/nc-gui/components/cell/Email.vue:89]: why `inline-block` is required here?

**Review by wingkwong — CHANGES_REQUESTED** [review]
Please run lint after all the changes.

File: packages/nc-gui/assets/style.scss (hunk 2 above)
Comment (reenphygeorge) [reply packages/nc-gui/assets/style.scss:37]: It is used to align the check-mark icon in ant-message

File: packages/nc-gui/components/cell/Email.vue (hunk 3 above)
Comment (reenphygeorge) [reply packages/nc-gui/components/cell/Email.vue:89]: The clickable area in email used to take up the entire cell. Here inline-block is used to make sure the link only takes up the space occupied by the email text.

File: packages/nc-gui/assets/style.scss (hunk 2 above)
Comment (wingkwong) [reply packages/nc-gui/assets/style.scss:37]: ok. please add the comment in the code then.

File: packages/nc-gui/components/cell/Email.vue (hunk 3 above)
Comment (wingkwong) [reply packages/nc-gui/components/cell/Email.vue]: got it.

**Review by dstala — APPROVED** [review]

**Pull request merged**